ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9ModelStruct.h

```diff
@@ -23,7 +23,6 @@
 #include <math.h>
 #include <stdlib.h>
 
-
 using namespace std;
 
 #define __GETR(x) ((x & 0x0000FF))
```

Comment:
We have this macro about 72x in our source. When you see it can you start removing it.

Reply:
Yes; I need to speak with you about this senior.